CODEOWNERS

```diff
@@ -43,6 +43,8 @@ homeassistant/components/climate/eq3btsmart.py @rytilahti
 homeassistant/components/climate/sensibo.py @andrey-git
 homeassistant/components/cover/template.py @PhracturedBlue
 homeassistant/components/device_tracker/automatic.py @armills
+homeassistant/components/media_player/kodi.py @armills
+homeassistant/components/media_player/monoprice.py @etsinko
```

Comment:
The first line is not needed.

Put the monoprice line under `homeassistant/components/media_player/kodi.py` below

Reply:
Yes, I noticed it too. Already fixed.